File: 1542.py
while True:
    e = [int(x) for x in str(input()).split()]
    if e[0] == 0: break
    q = e[0]
    d = e[1]
    p = e[2]
    n = int((q * d * p) / (p - q))
    if n == 1: print('1 pagina')
    else: print('{} paginas'.format(n))

File: 2-ad-hoc/1542.cpp
#include <iostream>

using namespace std;

int main(void) {
    int A, B, C, N;

    while (cin >> A && A != 0) {
        cin >> B >> C;
        N = int((A * B * C) / (C - A));
        if (N == 1) cout << "1 pagina" << endl;
        else cout << N << " paginas" << endl;
    }
    return 0;
}
